### ISSUE TITLE
Scene Name: exceptions keep disappearing

### ISSUE THREAD

**Issue by nydave69** [pymedusa/Medusa#10550]
**Describe the bug**
I add exceptions,   after a few days I have to do it again because Medusa says it not in the list...
I add it again,  Medusa will then be able to match the downloads.

It's totally random shows...  it's a daily occurrence...

Also I have had Medusa loose some of it's settings,  a restart will reload them.  but not the scene exceptions.
Nothing in the logs..  One time it couldn't find the download directory, it had disappeared in settings.

**To Reproduce**

Can't reproduce manually,  have to let it run and see when errors start showing up.  mostly strange stuff.

**Expected behavior**

**Screenshots**

**Medusa (please complete the following information):**
<html>
<body>

Medusa Info: | Branch: masterCommit: e4870f8f4a322e67c77434597ccc09aa6524aa05Version: 0.5.29Database: 44.19
-- | --
Python Version: | 3.7.3 (v3.7.3:ef4ec6ed12, Mar 25 2019, 22:22:05) [MSC v.1916 64 bit (AMD64)]
SSL Version: | OpenSSL 1.1.0j 20 Nov 2018
OS: | Windows-7-6.1.7601-SP1
Locale: | en_US.cp1252
Timezone: | Eastern Daylight Time
  |  
  |  
User: | Plex
Program Folder: | C:\Medusa\Medusa
Config File: | C:\Medusa\Data\config.ini
Database File: | C:\Medusa\Data\main.db
Cache Folder: | C:\Medusa\Data\cache
Log Folder: | C:\Medusa\Data\Logs
Arguments: | --nolaunch --datadir=C:\Medusa\Data

</body>
</html>

**Debug logs (at least 50 lines):**
General > Advanced Settings > Enable debug
<details>

```
<-- Please replace this whole line with your debug logs -->
```
</details>

**Additional context**
Medusa Info:	Branch: [master](https://github.com/pymedusa/Medusa/tree/master)
Commit: [e4870f8f4a322e67c77434597ccc09aa6524aa05](https://github.com/pymedusa/Medusa/commit/e4870f8f4a322e67c77434597ccc09aa6524aa05)
Version: [0.5.29](https://github.com/pymedusa/Medusa/releases/tag/v0.5.29)
Database: 44.19
 Python Version:	3.7.3 (v3.7.3:ef4ec6ed12, Mar 25 2019, 22:22:05) [MSC v.1916 64 bit (AMD64)]
 SSL Version:	OpenSSL 1.1.0j 20 Nov 2018
 OS:	Windows-7-6.1.7601-SP1
 Locale:	en_US.cp1252
 Timezone:	Eastern Daylight Time
 	 
 	 
 User:	Plex
 Program Folder:	C:\Medusa\Medusa
 Config File:	C:\Medusa\Data\config.ini
 Database File:	C:\Medusa\Data\main.db
 Cache Folder:	C:\Medusa\Data\cache
 Log Folder:	C:\Medusa\Data\Logs
 Arguments:	
--nolaunch --datadir=C:\Medusa\Data

**Comment by p0psicles** [comment]
I have to ask just in case. When you add a scene exception. Do you also add the `+` icon and save after?

**Comment by nydave69** [comment]
yes...   I'll add 2 or 3 and a week later I'll get a message to add a scene exception for it again
There a no exceptions listed for that show.

Not sure if this is related but I have noticed settings are disappearing..
They are in the config file but if I look via the GUI it's missing, a restart and it's back.

**Comment by p0psicles** [comment]
I don't know what to say man. You'r really the only user with these kind of issues. And we have thousands. Most often this has something to do with file access/rights. But your saying that the config.ini shows the data just fine. So that also really can't be it.

If you've verified that the config.ini has the correct data. Then you could try analyzing the network calls in the browser's dev console. To see what the /api/v2/* calls are returning. Those should eventually result in the settings being displayed in the UI.

So then i'm out of ideas. You don't have two instances running using the same config by chance?
Also which browser are you using.

And last i noticed you're using the default windows encoding. `en_US.cp1252`.
Try setting the environment variable PYTHONUTF8=1 on your Windows machine. Maybe it'll help.

**Comment by nydave69** [comment]
where are scene exception stored?

UTF8 encoding, tried all the options listed here and on googling how to set it..  none have worked.
There is a short explanation in the Python group as to why they do not recommend setting it in windows or globally.
Python says developers should set the required code page at their application start up. So that applications needing different code page can run on the same system.

**Comment by p0psicles** [comment]
Scene exceptions are stored in main.db.

Can you link me to that doc?
